src/Microsoft.Kusto.ServiceLayer/DataSource/KustoClient.cs

```diff
@@ -161,28 +161,39 @@ public IDataReader ExecuteQuery(string query, CancellationToken cancellationToke
             IDataReader[] origReaders = new IDataReader[script.Blocks.Count];
             try
             {
+                var numOfQueries = 0;
```

Comment:
Isn't the number of queries just the length of `script.Blocks`?

Reply:
Not really, even comments are considered as blocks in Kusto api